csrc/codegen.cpp

```diff
@@ -2995,9 +2987,17 @@ class CudaKernelGenerator : private kir::ConstIrVisitor {
           break;
         case MemoryType::Local: {
           auto va = kernel_->summary().vectorized_accesses;
+          unsigned int vect_factor = 0;
           if (va.find(tv) != va.end()) {
+            vect_factor = va.at(tv);
+          } else if (
+              kernel_->summary().num_grouped_iterations > 1 &&
+              ir_utils::isConsumedByIterGroupedReduction(tv)) {
+            vect_factor = kernel_->summary().num_grouped_iterations;
+          }
+          if (vect_factor > 0) {
```

Comment:
When can this be 0?

Reply:
It is initilized to 0, so stays at 0 when the tv is not vectorized with gmem or smem.